flexget/components/notify/notifiers/cronitor.py

```diff
@@ -0,0 +1,91 @@
+from builtins import *
+
+import logging
+import socket
+from flexget import plugin
+from flexget.event import event
+from flexget.plugin import PluginWarning
+from requests.exceptions import RequestException
+from flexget.utils.requests import Session as RequestSession
+
+requests = RequestSession(max_retries=3)
+
+plugin_name = "cronitor"
+
+log = logging.getLogger(plugin_name)
+
+
+class Cronitor(object):
+    """
+    Example::
+      cronitor:
+        monitor_code: ABC123
+    """
+
+    base_url = "https://cronitor.link/{monitor_code}/{status}"
+
+    schema = {
+        "oneOf": [
+            {
+                "type": "object",
+                "properties": {
+                    "monitor_code": {"type": "string"},
+                    "on_start": {"type": "boolean"},
+                    "on_abort": {"type": "boolean"},
+                    "message": {"type": "string"},
+                    "host": {"type": "string"},
+                    "auth_key": {"type": "string"},
+                },
+                "required": ["monitor_code"],
+                "additionalProperties": False,
+            },
+            {"type": "string"},
+        ]
+    }
+
+    def __init__(self):
+        self.config = None
```

Comment:
This is not good practice as it leaves (possibly) behind all kinds of objects after plugin execution has finished.

Reply:
it's either this or prepare config and pass it for every task execution phase. any recommendation?